core/src/setup/Kokkos_Setup_HIP.hpp

```diff
@@ -58,8 +58,8 @@
 
 #define KOKKOS_IMPL_FORCEINLINE_FUNCTION __device__ __host__ __forceinline__
 #define KOKKOS_IMPL_INLINE_FUNCTION __device__ __host__ inline
-#define KOKKOS_DEFAULTED_FUNCTION __device__ __host__ inline
-#define KOKKOS_INLINE_FUNCTION_DELETED __device__ __host__ inline
+#define KOKKOS_DEFAULTED_FUNCTION __device__ __host__
+#define KOKKOS_INLINE_FUNCTION_DELETED __device__ __host__
```

Comment:
Do we need the `__host__ __device__` annotation in the first place?

Reply:
Let's see what 4093f63bc shows.